mobile/src/navigation/app/common/constants.ts

```diff
@@ -132,6 +135,11 @@ const NAVIGATION_ITEMS: DrawerNavigationList[] = [
         component: EditCourseCategory,
         permissions: [PermissionKey.MANAGE_CATEGORIES],
       },
+      {
```

Comment:
Now, as we added new navigation, EDIT_COURSE_CATEGORY, COURSE and CHOOSE_A_MENTOR are no longer in app navigation,so you need to move them and logic related to them to course navigation

Reply:
Also don't forget about permissions

Reply:
Sorry, don't move EDIT_COURSE_CATEGORY, it has another type of navigation